Platform Documentation.md

```diff
@@ -854,31 +854,31 @@ $ cctrlapp APP_NAME/DEP_NAME deploy --stack [luigi,pinky]
 ~~~
 
 [generating SSH keys]: https://help.github.com/articles/generating-ssh-keys
-[Custom Config Add-on]: https://www.cloudcontrol.com/dev-center/Add-on%20Documentation/Deployment/Custom%20Config
+[Custom Config Add-on]: https://www.cloudcontrol.com/dev-center/add-on-documentation/custom-config
 [web console]: https://www.cloudcontrol.com/console
 [API libraries]: https://github.com/cloudControl
-[the latest version]: https://www.cloudcontrol.com/download/win
+[the latest version]: http://download.cloudcontrolled.com/windows
 [Python 2.6+]: http://python.org/download/
 [reset your password]: https://api.cloudcontrol.com/reset_password/
 [quick Git tutorial]: http://rogerdudler.github.com/git-guide/
 [Bazaar in five minutes]: http://doc.bazaar.canonical.com/latest/en/mini-tutorial/
 [Heroku buildpack API]: https://devcenter.heroku.com/articles/buildpack-api
-[guides]: https://www.cloudcontrol.com/dev-center/Guides
+[guides]: https://www.cloudcontrol.com/dev-center/guides
 [MongoLab Add-on]: https://www.cloudcontrol.com/add-ons/mongolab
 [Add-on marketplace]: https://www.cloudcontrol.com/add-ons
-[Deployment category]: https://www.cloudcontrol.com/dev-center/Add-on%20Documentation/Deployment
+[Deployment category]: https://www.cloudcontrol.com/dev-center/add-on-documentation/deployment
 [rsyslog]: http://www.rsyslog.com/
 [TLS]: http://en.wikipedia.org/wiki/Transport_Layer_Security
 [Alias Add-on]: https://www.cloudcontrol.com/add-ons/alias
-[Blitz.io]: https://www.cloudcontrol.com/dev-center/Add-on%20Documentation/Performance%20&%20Monitoring/Blitz.io
+[Blitz.io]: https://www.cloudcontrol.com/dev-center/add-on-documentation/blitz.io
```

Comment:
This needs to be `/dev-center/add-on-documentation/blitz` please also see https://github.com/cloudControl/website/issues/213